odoo/sql_db.py

```diff
@@ -468,6 +470,14 @@ def __getattr__(self, name):
     def closed(self):
         return self._closed
 
+    def now(self):
+        try:
+            return self._now
+        except AttributeError:
+            self.execute("SELECT (now() AT TIME ZONE 'UTC')")
+            self._now = self.fetchone()[0]
+            return self._now
+
```

Comment:
We could think about it, but its current API does not give that function access to the cursor object...

Reply:
Maybe we could have a placeholder / somewhat magic value which translates to `now() as time zone utc` when mogrified? Because having to query the DB just so we can send the same value back in is a bit odd.

Reply:
@xmo-odoo the point is to put the value it in memory (record cache) even before sending it to the database as part of an UPDATE.

Reply:
BTW this avoids flushing the pending updates when all you want is get `write_date` after a call to `write()`.

Reply:
Note that SQL's `now()` is transactional and will always return the timestamp of the transaction start, whereas `fields.Datetime.now()` is the wall time. Those 2 behaviors are desirable for different reasons, in different cases.